server/services/interventionsService.test.ts

```diff
@@ -0,0 +1,50 @@
+import { pactWith } from 'jest-pact'
+import InterventionsService from './interventionsService'
+import config from '../config'
+
+pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, provider => {
+  let interventionsService: InterventionsService
+
+  beforeEach(() => {
+    const testConfig = { ...config.apis.interventionsService, url: provider.mockService.baseUrl }
+    interventionsService = new InterventionsService(testConfig)
+  })
+
+  describe('getReferral', () => {
+    beforeEach(async () => {
+      // This is just an example for getting Pact set up; we’ll properly design this endpoint later
+      await provider.addInteraction({
+        state: 'There is an existing referral with ID of 1',
+        uponReceiving: 'a request for that referral',
+        withRequest: {
+          method: 'GET',
+          path: '/referrals/1',
+          headers: { Accept: 'application/json' },
+        },
+        willRespondWith: {
+          status: 200,
+          body: {
+            id: '1',
+            desiredOutcomes: [
+              {
+                id: '1',
```

Comment:
in the example code i read, these were like matcher objects, which looked to basically check the structure of the returned data, not nessecarily the values. is this how this works too?

Reply:
I've converted this to use `Matcher.like` so the value isn't as important, which will be more useful once the contract artifacts are generated.